test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14225,7 +14225,12 @@ def create_eatcpu_job(self, duration=None, mom=None):
         """
         script_dir = os.path.dirname(os.path.dirname(__file__))
         script_path = os.path.join(script_dir, 'utils', 'jobs', 'eatcpu.py')
-        if mom:
+        if not DshUtils().is_localhost(mom):
+            pbs_conf = DshUtils().parse_pbs_config(mom)
+            shell_path = os.path.join(pbs_conf['PBS_EXEC'],
+                                   'bin', 'pbs_python')
+            a = {ATTR_S: shell_path}
+            self.set_attributes(a)
```

Comment:
this should be outside of `if not localhost` condition right? because we want to use that shell as python whether mom is remote or not, right?

Reply:
yes, thanks for catching that i will make the change